### DISCUSSION TITLE
Hotkeys for sorting modes

### DISCUSSION THREAD

**Discussion by maanemann** [gasparschott/obsidian-continuous-mode#25] [Ideas]
## Idea

Hotkeys for sorting modes to make a mouseless workflow.

## Background

This plugin is ideal for displaying all my daily notes in what seems to be one file. I create one every day for today's tasks and notes.

The problem is that the oldest one is on top by default, as the file names are dates, and 1 comes bedore 2 alphabetically (which is the default sorting mode). To fix that i have to find a menu item in a sub-sub-context menu. Really slow for such a common action!

One solution would be an option to bind sorting options to hotkeys, like you can with most other options (highly appreciated). This way you could enable the desired continuous mode and then sort it as prefered, all with two hotkeys.

Then combined with the existing option to limit amount of files to open simultaneously, this would be a perfect addition to the daily workflow!

![image](https://github.com/user-attachments/assets/1674da7a-104c-4990-bb67-39ed37552abc)

**Comment by gasparschott** [comment]
Thanks, good idea! I’ll add the commands in the next release.

**Comment by maanemann** [comment]
Awesome, i look forward to it! Another solution would be to have a default sorting selection in the settings. The two features could exist well together.
